tests/test_read.py

```diff
@@ -252,6 +253,21 @@ def test_check_conforming_no_well():
     assert not las.check_conforming()
 
 
+def test_check_conforming_no_curves():
+    las = lasio.read(egfn("missing_curves_section.las"))
+    assert not las.check_conforming()
+
+
+def test_check_conforming_no_ascii():
+    las = lasio.read(egfn("missing_ascii_section.las"))
+    assert not las.check_conforming()
+
+
+def test_check_conforming_no_ascii_2():
+    las = lasio.read(egfn("missing_curves_section.las"))
+    assert not spec.AsciiSectionExists.check(las)
```

Comment:
?? This test should be failing: `missing_curves_section.las` contains an ~A section.

Reply:
I added this to reflect current behavior of `spec.AsciiSectionExists.check`, where ascii sections existence is checked through existence of data in Curves section only.
So if the Curves section is missing, there is no way to access the data or ascii section.
This ties back to your earlier comment where you want to support reading files like `barebones2.las` where only data sections exist.

Reply:
I can delete the test and we can add something similar later